web/src/pages/addDevice/steps/AddDeviceConfigStep/AddDeviceConfigStep.tsx

```diff
@@ -78,14 +78,20 @@ export const AddDeviceConfigStep = () => {
       <div className="info">
         <p>{localLL.qrInfo()}</p>
       </div>
-      <DeviceConfigsCard
-        deviceId={device.id}
-        publicKey={publicKey}
-        privateKey={privateKey}
-        userId={userData.id}
-        networks={networks}
-        deviceName={device.name}
-      />
+      {networks.length ? (
```

Comment:
Do not use ? : syntax in templates.
Do not use implicit bool assertions in templates, it can result in rendering "0" or some string instead of a component.